packages/cli/src/databases/repositories/AbstractRepository.ts

```diff
@@ -0,0 +1,65 @@
+import { Repository } from 'typeorm';
+import type { WorkflowEntity } from '../entities/WorkflowEntity';
+import { jsonParse } from 'n8n-workflow';
+import type { JsonValue } from 'n8n-workflow';
+import * as utils from '@/utils';
+import { validate as jsonSchemaValidate } from 'jsonschema';
+import type { Constructor } from '@/Interfaces';
+
+type ParsedJSON = JsonValue;
+
+function mixinQueryMethods<T extends Constructor<{}>>(base: T) {
+	class Derived extends base {
+		static toQueryFilter(rawFilter: string, schema: { properties: object }) {
```

Comment:
maybe when parsing the raw query string, we should switch over to using `class-transformer` and `class-validator` to have stronger compile time and runtime typechecks.

Reply:
08d66e9